### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by lee-brian**
Include more info on how monitors are tagged to services

### Merge instructions

- [ x] Please merge after reviewing

### CODE REVIEW

File: content/en/service_catalog/guides/troubleshooting.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you have many services that share the same metadata, you do not need separate
 
 ## Associated monitors not displayed in the Setup Guidance section
 
-The Service Catalog associates monitors to services when you tag the monitor with `service` and [APM primary tags][3]. 
+The Service Catalog associates monitors to services when are tagged, scoped or grouped with service or [APM primary tags][3].
```
Comment (rtrieu) [inline content/en/service_catalog/guides/troubleshooting.md:38]:
```suggestion
The Service Catalog associates monitors to services when they are tagged, scoped, or grouped with service or [APM primary tags][3]. 
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback but otherwise looks good!

**Review by lee-brian — COMMENTED** [review]
Looks good! Thank you

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**